.header.md

```diff
@@ -55,13 +55,19 @@ This deployment uses `ubuntu-focal-20.04-amd64-server` AWS AMI. It will **not**
 
 If, for some reason, base AMI is required to get updated, 
 it can be achieved by running `terraform taint` command for each instance, before `terraform apply`.   
-Instances can be tainted by running the `terraform taint module.instances[<AZ>].aws_instance.polygon_edge_instance` command, 
-where `<AZ>` is the availability zone
-Example with default configuration:
+Instances can be tainted by running the `terraform taint module.instances[<instance_number>].aws_instance.polygon_edge_instance` command.
+
+Example:
 ```shell
-terraform taint module.instances[\"us-west-2a\"].aws_instance.polygon_edge_instance
-terraform taint module.instances[\"us-west-2b\"].aws_instance.polygon_edge_instance
-terraform taint module.instances[\"us-west-2c\"].aws_instance.polygon_edge_instance
-terraform taint module.instances[\"us-west-2d\"].aws_instance.polygon_edge_instance
+terraform taint module.instances[\"0\"].aws_instance.polygon_edge_instance
+terraform taint module.instances[\"1\"].aws_instance.polygon_edge_instance
+terraform taint module.instances[\"2\"].aws_instance.polygon_edge_instance
+terraform taint module.instances[\"3\"].aws_instance.polygon_edge_instance
```

Comment:
```suggestion
terraform taint module.instances[0].aws_instance.polygon_edge_instance
terraform taint module.instances[1].aws_instance.polygon_edge_instance
terraform taint module.instances[2].aws_instance.polygon_edge_instance
terraform taint module.instances[3].aws_instance.polygon_edge_instance
```

Reply:
Great catch. Key used to be of string type, and I forgot to delete `" "`. Fixed in cb7044370229b899ed7d9cd4831705b9108f30e3